service/dps/storm/topologies/validation/src/test/java/eu/europeana/cloud/service/dps/storm/topologies/validation/topology/ValidationTopologyTest.java

```diff
@@ -9,6 +9,7 @@
 import eu.europeana.cloud.service.dps.storm.NotificationTuple;
 import eu.europeana.cloud.service.dps.storm.ParseTaskBolt;
 import eu.europeana.cloud.service.dps.storm.io.*;
+import eu.europeana.cloud.service.dps.storm.topologies.validation.topology.bolts.StatisticsBolt;
 import eu.europeana.cloud.service.dps.storm.topologies.validation.topology.bolts.ValidationBolt;
```

Comment:
The ValidationExecutionService has been deployed to the artifactory, and this changes include removing the default -with no parameters- constructor. So for now If I want to clean install -as I did- , my build will fail. and I can't run ValidationTopologyTest tests.